tests/cpp/operator/batchnorm_test.cc

```diff
@@ -109,67 +106,67 @@ class BNOperatorExecutor : public test::op::LegacyOperatorExecutor<DType, AccRea
 
     // Init the moving data (all mean = 0, all var = 1)
     MSHADOW_TYPE_SWITCH(
-      this->c_.blob_aux_states_[mxnet::op::batchnorm::kMovingMean].type_flag_,
+      this->input_blobs()[mxnet::op::batchnorm::kMovingMean].type_flag_,
```

Comment:
I don;t know what vectors you use for what in your API, so these would need to be matched up correctly.
Probably implementing getBlob(enum data type such as fwd input grad) in the batch norm executor would probably be the easiest way